apps/src/gamelab/AnimationFrameList.jsx

```diff
@@ -0,0 +1,26 @@
+/** @file Vertical scrolling list of animation frames. */
+'use strict';
+
+var AnimationFrameTile = require('./AnimationFrameTile.jsx');
+var NewItemTile = require('./NewItemTile.jsx');
+var ScrollableList = require('./ScrollableList.jsx');
+
+/**
+ * Vertical scrolling list of animation frames for the selected animation.
+ * This component should be unique within the animation tab.
+ */
+var AnimationFrameList = React.createClass({
+  render: function () {
+    return (
+      <ScrollableList className="animation-frame-list">
+        <AnimationFrameTile index={1} />
+        <AnimationFrameTile index={2} />
+        <AnimationFrameTile index={3} isSelected />
+        <AnimationFrameTile index={4} />
+        <AnimationFrameTile index={151} />
+        <NewItemTile label="new frame" />
+      </ScrollableList>
+    );
+  }
+});
+module.exports = AnimationFrameList;
```

Comment:
Questions:

1. Do we prefer functional stateless components for any pure component?  That wasn't totally clear to me from the styleguide.

2. Is that worth doing here, where I expect this component to expand in the very near future and no longer be simple/stateless?